easybuild/easyconfigs/m/MUSCLE3-Python/MUSCLE3-Python-0.5.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'PythonPackage'
+
+name = 'MUSCLE3-Python'
+version = '0.5.0'
+
+homepage = 'https://muscle3.readthedocs.io/en/latest/index.html'
+
+description = """MUSCLE3 allows connecting multiple simulation models
+ together into a multiscale simulation. Simulation models can be as
+ simple as a single Python file, or as complex as a combination of
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

Comment:
@SimonPinches missing build deep for `binutils`, as added to the others.

Reply:
@verdurin Changed to use foss toolchain so not needed now.